mlflow/gateway/utils.py

```diff
@@ -111,16 +127,16 @@ def decode(cls, encoded_token: str):
         try:
             decoded_token = base64.b64decode(encoded_token)
             parsed_token = json.loads(decoded_token)
+            index = int(parsed_token.get("index"))
```

Comment:
adding the int casting to handle invalid continuation tokens

Reply:
awesome - thank you :)